.github/dependabot.yml

```diff
@@ -19,7 +19,7 @@ updates:
       time: "11:00"
     ignore:
       - dependency-name: "@types/node"
-        versions: ["15.x", "14.x", "13.x"]
+        versions: ["16.x", 15.x", "14.x", "13.x"]
```

Comment:
wonder if it's better to ignore all major version updates? https://github.blog/changelog/2021-05-21-dependabot-version-updates-can-now-ignore-major-minor-patch-releases/

Reply:
Definitely! TIL! I'll change now